### ISSUE TITLE
Issues running tests on the various packages.

### ISSUE THREAD

**Issue by IngwiePhoenix** [soyuka/relieve#6]
Hello!

I wanted to see if this package works properly by running it's test suite and, some tests do work, some do not. Here is the whole output I got.

I noticed that the packages all require through `relieve/...`. Maybe I oversaw how to set up the testing somewhere?

Kind regards,
Ingwie

```
C:\Users\ingwi\AppData\Local\Temp\node\relieve>npm i
added 136 packages from 109 contributors and audited 202 packages in 1.832s
found 4 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

C:\Users\ingwi\AppData\Local\Temp\node\relieve>npm test

> relieve@2.2.1 test C:\Users\ingwi\AppData\Local\Temp\node\relieve
> lerna run test

Lerna v2.0.0-beta.38

> relieve-logger@2.2.3 test C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-logger
> mocha --exit



  RelieveLogger
    √ should create log file and close stream on exit
    √ should append to log file
    √ should rotate delay
    √ should not rotate delay
    √ should rotate size
    √ should not rotate size


  6 passing (15ms)



> relieve@2.2.3 test C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve
> mocha



  utils
    defineNameProperty
      √ should register name getter/setter
      √ should get default name
      √ should fail setting name twice

  strategies
    Strategy
      √ should add some values
      √ should get the next value
      √ should get the next value
      √ should get the next value
      √ should get test1
      √ should get nothing
      √ should remove strategy

  task
    ForkTask
      √ should throw because ipc is not available
      √ should create a new without constructor
      √ should create a new task
      √ should get message through task (46ms)
      √ should not be available because child has been killed
    ScriptTask
      √ should fail creating a new ScriptTask
      √ should create a new ScriptTask (59ms)
      √ should reject starting again
      √ should restart (64ms)
      √ should reject start
      √ should kill
      √ should create an autorestart task (62ms)
      √ should register event before starting (58ms)
      √ should send a message to the task and resolve promise when message has been delivered
      √ should fail sending a message if the task is not started
      √ should start a task with arguments (59ms)
      √ should start a task with complex arguments (57ms)
      √ should add one interface
      √ should not add invalid interface
      √ should stop and not restart (63ms)
      √ should still be an autorestart task (63ms)
    CallableTask
      √ should create a new CallableTask (59ms)
      √ should create an autorestart callable task (63ms)
      √ should create a new CallableTask without constructor (59ms)
      √ should call a function
      √ should have register an emiting interval
      √ should stop the interval
      √ should get the value
      √ should get the args
      √ should throw an get error event
      √ should create a new CallableTask with a different Container (59ms)
      √ should kill the task
      √ should be monitorable (60ms)

  worker
    Worker
      √ should add a task
      √ should add a second task and get them all
      √ should send a message to every task
      √ should remove task
      √ should send SIGINT to tasks
      √ should not be able to set tasks
    QueueWorker
      √ should fail adding a ForkTask
      √ should add and remove a task
      √ should add three tasks
      √ should start one task (264ms)
      √ should start every task in series (523ms)
      √ should start every tasks in concurrency
    CloudWorker
      √ should fail adding a ForkTask
      √ should add and remove a task
      √ should add and run two tasks (62ms)
      √ should add and start a third one (58ms)
      √ should receive one answer
      √ should get one answer
      √ should call
      √ should exit worker
      √ should reject the promise because task is not callable (59ms)


  64 passing (2s)


NpmUtilities.execInDir        ("run test", [], "C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe")
Error: Command failed: npm run test
C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1163
      else throw err
           ^

Error: Cannot find module 'relieve/tasks/ScriptTask'
Require stack:
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\one-and-dones.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\options.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:311:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:308:14)
    at Mocha.run (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:849:10)
    at Object.exports.singleRun (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:108:16)
    at exports.runMocha (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:143:13)
    at Object.exports.handler (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run.js:305:3)
    at Object.runCommand (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\lib\command.js:242:26)
    at Object.parseArgs [as _parseArgs] (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1087:28)
    at Object.parse (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:566:25)
    at Object.exports.main (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\cli.js:68:6)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha:133:29)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\end2end.js',
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\index.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\mocha.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\one-and-dones.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\options.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\bin\\mocha'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! relieve-failsafe@2.2.5 test: `mocha -b test/index.js --exit`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the relieve-failsafe@2.2.5 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ingwi\AppData\Roaming\npm-cache\_logs\2020-02-05T10_27_13_929Z-debug.log

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:321:20)
    at maybeClose (internal/child_process.js:1026:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)

> relieve-failsafe@2.2.5 test C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe
> mocha -b test/index.js --exit


NpmUtilities.runScriptInDir   ("test", [], "C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe")
Error: Command failed: npm run test
C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1163
      else throw err
           ^

Error: Cannot find module 'relieve/tasks/ScriptTask'
Require stack:
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\one-and-dones.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\options.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:311:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:308:14)
    at Mocha.run (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:849:10)
    at Object.exports.singleRun (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:108:16)
    at exports.runMocha (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:143:13)
    at Object.exports.handler (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run.js:305:3)
    at Object.runCommand (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\lib\command.js:242:26)
    at Object.parseArgs [as _parseArgs] (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1087:28)
    at Object.parse (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:566:25)
    at Object.exports.main (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\cli.js:68:6)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha:133:29)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\end2end.js',
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\index.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\mocha.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\one-and-dones.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\options.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\bin\\mocha'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! relieve-failsafe@2.2.5 test: `mocha -b test/index.js --exit`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the relieve-failsafe@2.2.5 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ingwi\AppData\Roaming\npm-cache\_logs\2020-02-05T10_27_13_929Z-debug.log

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:321:20)
    at maybeClose (internal/child_process.js:1026:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)

> relieve-failsafe@2.2.5 test C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe
> mocha -b test/index.js --exit



> relieve-failsafe@2.2.5 test C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe
> mocha -b test/index.js --exit


Errored while running npm script 'test' in 'relieve-failsafe'
Error: Command failed: npm run test
C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1163
      else throw err
           ^

Error: Cannot find module 'relieve/tasks/ScriptTask'
Require stack:
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\one-and-dones.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\options.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:311:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:308:14)
    at Mocha.run (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:849:10)
    at Object.exports.singleRun (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:108:16)
    at exports.runMocha (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:143:13)
    at Object.exports.handler (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run.js:305:3)
    at Object.runCommand (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\lib\command.js:242:26)
    at Object.parseArgs [as _parseArgs] (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1087:28)
    at Object.parse (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:566:25)
    at Object.exports.main (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\cli.js:68:6)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha:133:29)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\end2end.js',
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\index.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\mocha.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\one-and-dones.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\options.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\bin\\mocha'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! relieve-failsafe@2.2.5 test: `mocha -b test/index.js --exit`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the relieve-failsafe@2.2.5 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ingwi\AppData\Roaming\npm-cache\_logs\2020-02-05T10_27_13_929Z-debug.log

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:321:20)
    at maybeClose (internal/child_process.js:1026:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
Errored while running RunCommand.execute
Error: Command failed: npm run test
C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1163
      else throw err
           ^

Error: Cannot find module 'relieve/tasks/ScriptTask'
Require stack:
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js
- C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\one-and-dones.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\options.js
- C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\end2end.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Local\Temp\node\relieve\packages\relieve-failsafe\test\index.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:311:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:308:14)
    at Mocha.run (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\mocha.js:849:10)
    at Object.exports.singleRun (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:108:16)
    at exports.runMocha (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run-helpers.js:143:13)
    at Object.exports.handler (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\run.js:305:3)
    at Object.runCommand (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\lib\command.js:242:26)
    at Object.parseArgs [as _parseArgs] (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:1087:28)
    at Object.parse (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\node_modules\yargs\yargs.js:566:25)
    at Object.exports.main (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\lib\cli\cli.js:68:6)
    at Object.<anonymous> (C:\Users\ingwi\AppData\Roaming\npm\node_modules\mocha\bin\mocha:133:29)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\end2end.js',
    'C:\\Users\\ingwi\\AppData\\Local\\Temp\\node\\relieve\\packages\\relieve-failsafe\\test\\index.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\mocha.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\one-and-dones.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\lib\\cli\\options.js',
    'C:\\Users\\ingwi\\AppData\\Roaming\\npm\\node_modules\\mocha\\bin\\mocha'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! relieve-failsafe@2.2.5 test: `mocha -b test/index.js --exit`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the relieve-failsafe@2.2.5 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ingwi\AppData\Roaming\npm-cache\_logs\2020-02-05T10_27_13_929Z-debug.log

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:321:20)
    at maybeClose (internal/child_process.js:1026:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
npm ERR! Test failed.  See above for more details.
```

**Comment by soyuka** [comment]
Hi ! Thanks for reporting this, I just ran the tests and indeed there was a small issue fixed in 
https://github.com/soyuka/relieve/commit/22c68d6a2bacfd54b91befe5ba15be540ef42b24

To run the tests make sure to run:

```
lerna bootstrap
npm run test
```
